src/Core/View.js

```diff
@@ -761,6 +766,10 @@ class View extends THREE.EventDispatcher {
      * @return {number} The projected distance in meters.
      */
     getPixelsToMeters(pixels = 1, screenCoord) {
+        if (this.camera.camera3D.isOrthographicCamera) {
+            screenMeters = (this.camera.camera3D.right - this.camera.camera3D.left) / this.camera.camera3D.zoom;
```

Comment:
(this.camera.camera3D.right - this.camera.camera3D.left) equals  `this.camera.width`?